UI/signup.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+    	<meta name="viewport" content="width=device-width,initial-scale=1.0">
+		<title>politico : Sign-up</title>
+		<link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+		<link href="assets/css/styles.css" type="text/css" rel="stylesheet" media="all" />
+		<link href="assets/css/signup.css" type="text/css" rel="stylesheet" media="all" />
+		<link rel="shortcut icon" type="image/x-icon" href="./assets/images/logo.gif">
+	</head>
+	<body>
+			<nav class="politico-nav">
+					<div class="nav-container">
+						<div class="nav-left">
+							<a href="./frontPage.html"><h2><img class="laravel" src="./assets/images/logo.gif" height="40px" width="40px" alt="">  POLITICO</h2></a>
+						</div>
+		
+						<div class="nav-right">
+											
+						<ul class="header">
+								<li class="login-btn">
+									<a href="./Application.html">Apply</a> 
+								</li>
+								<li class="signup-btn"> 
+								   <a href="./signin.html">sign in</a> 
+								</li>
+								<li>
+										<a href="./signin.html">
+											<button class="nav-button">Logout</button>
+										</a>
+								</li>
+							</ul>
+		
+							
+						</div>
+					</div>
+		
+					
+		
+				</nav>
+		<div class="signup-area">
+
+			<div class="signup-box">
+				<h4 class="signup-box-title">Sign-up</h4>
+				<form action="./UserProfile.html">
+					<div class="signup-input-section">
+						<input type="text" placeholder="First Name"/>
+					</div>
+					<div class="signup-input-section">
+						<input type="text" placeholder="Last Name"/>
+					</div>
+					<div class="signup-input-section">
+						<input type="email" placeholder="Email"/>
+					</div>
+					<div class="signup-input-section">
+						<input type="password" placeholder="Password"/>
+					</div>
+					<div class="signup-input-section">
+						<input type="password" placeholder="Retype Password"/>
+					</div>
+					<div class="signup-submit">
+						<button type="submit">Submit</button>
+					</div>
+				</form>
+				<div class="signup-box-footer">
+					<small>
+						<a href="./signin.html">Aleady have an account? &amp; Sign In</a>
+					</small>
+				</div>
+			</div>
+		</div>
+
+		<footer class="politico-footer">
+				<p>politico. &copy; 2019</p>
+			</footer>
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks